3주차/[Lv.3]_야근_지수/안혜수.js

```diff
@@ -0,0 +1,17 @@
+function solution(n, works) {
+    if(works.reduce((acc,cur) => acc + cur,0)<n)return 0;
+    if(works.length === 1)return (works[0]-n)**2;
+    works.sort((a,b) => b-a);
+    while(n){
+        works[0]-=1;
+        for(let i = 0 ; i < works.length; i++){
+            if(works[i]<works[i+1]){
+                let min = works[i];
```

Comment:
```suggestion
                const min = works[i];
```
`const`로 선언하는 게 더 좋을 것 같습니당 !